### DISCUSSION TITLE
Nested Form Alternative Configuration

### DISCUSSION THREAD

**Discussion by jaken99** [react-hook-form/react-hook-form#9323] [Q&A]
I am running into an issue with nested forms on the project I am currently working on. The user interface requires a form to be nested within a larger form, and the two forms are not related and their submissions/validations need to be handled separately. I am aware that nested forms are not directly supported, but is there any alternative configuration that might satisfy these requirements? It could be accomplished by triggering a modal containing the internal form outside the context of the larger form, but this would make the requested ui impossible to implement.

**Comment by bluebill1049** [comment]
This may solve your problem: https://github.com/react-hook-form/react-hook-form/issues/566#issuecomment-675515940 you can visually nest your forms by creating two useForm instances but not a form tag within a form tag, then use a state management lib to host/keep your two forms values.
